src/morphology/english/getVerbForms.js

```diff
@@ -0,0 +1,217 @@
+// "use strict";
+const irregularVerbs = require( "./irregularVerbs.js" );
+const verbPrefixesRegex = require( "./regexVerb.js" ).verbPrefixes;
+const sFormToInfinitiveRegex = require( "./regexVerb.js" ).sFormToInfinitive;
+const ingFormToInfinitiveRegex = require( "./regexVerb.js" ).ingFormToInfinitive;
+const edFormToInfinitiveRegex = require( "./regexVerb.js" ).edFormToInfinitive;
+const infinitiveToSFormRegex = require( "./regexVerb.js" ).infinitiveToSForm;
+const infinitiveToIngFormRegex = require( "./regexVerb.js" ).infinitiveToIngForm;
+const infinitiveToEdFormRegex = require( "./regexVerb.js" ).infinitiveToEdForm;
+
+const isUndefined = require( "lodash/isUndefined.js" );
+const unique = require( "lodash/uniqBy" );
+
+const normalizePrefixed = function( word ) {
```

Comment:
Could you add documentation to all functions in this file?

Reply:
Fixed.